README.md

```diff
@@ -1,75 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
 
-# Project Name
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-> Description the project.
+## Available Scripts
 
+In the project directory, you can run:
 
-## Built With
+### `npm start`
 
-- Major languages
-- Frameworks
-- Technologies used
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-## Live Demo (if available)
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
 
-[Live Demo Link](https://livedemo.com)
+### `npm test`
 
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-## Getting Started
+### `npm run build`
 
```

Comment:
- [ ] Please make to keep the Micoverse template and customize it to your own with your info